targets/csharp-unityv2/source/Shared/Internal/Log.cs

```diff
@@ -0,0 +1,39 @@
+using PlayFab;
+
+namespace PlayFab.Internal
+{
+    public class Log
+    {
+        public static void Debug(string text, params object[] args)
```

Comment:
We should seriously delete this.  This conflicts with how we'll be collecting logs later.